yb-voyager/src/dbzm/dbzm.go

```diff
@@ -64,12 +75,111 @@ func findDebeziumDistribution() error {
 	return nil
 }
 
+func GenerateAndStoreYBStreamID(config *Config) (string, error) {
+	err := findDebeziumDistribution(config.SourceDBType)
+	if err != nil {
+		return "", fmt.Errorf("error in finding debezium distribution: %s", err)
+	}
+	YB_CLIENT_WRAPPER_JAR := fmt.Sprintf("%s/yb-client-cdc-stream-wrapper.jar", DEBEZIUM_DIST_DIR) //$DEBEZIUM_DIST/yb-client-cdc-stream-wrapper.jar
+	tableName := strings.Split(config.TableList[0], ".")[1]                                        //any table name in the database is required by yb-client createCDCStream(...) API
+	command := fmt.Sprintf("java -jar %s -create -master_addresses %s -table_name %s -db_name %s ", YB_CLIENT_WRAPPER_JAR, config.YBServers, tableName, config.DatabaseName)
```

Comment:
Try limiting line length to below 100 columns.

You can add a line break after the format string and pass the arguments on the second line.